src/preload/syscallbuf.c

```diff
@@ -884,6 +884,29 @@ static int start_commit_buffered_syscall(int syscallno, void* record_end,
   return 1;
 }
 
+static void do_breakpoint(size_t value)
+{
+  char *unsafe_value = ((char*)-1)-0xf;
+  char *safe_value = (char*)&unsafe_value;
```

Comment:
Why not just use null for `unsafe_value`? And why not make `safe_value` a `char**` to avoid the cast?

Reply:
NULL may be dereferencable if set up via personality(2) (though frankly the only use I know of is for exploits). As far as I know the value I chose here should always cause a fault.